src/components/RocketCards.js

```diff
@@ -0,0 +1,69 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { reservedRocket, cancelledReserve } from '../redux/rockets/rockets';
+import '../styles/RocketCards.css';
+
```

Comment:
- [ ] Please add tests for your project as required 

![image](https://user-images.githubusercontent.com/75126481/183020659-1a7793e4-904d-4f6c-9f81-4557fb349826.png)